### DISCUSSION TITLE
Unable to import obfuscated package into Databricks

### DISCUSSION THREAD

**Discussion by apd-esierra** [dashingsoft/pyarmor#1994] [Q&A]
I'm running PyArmor 9 and built my Wheel package following the Docs and tutorials. When executing the obfuscated Wheel in my local machine it runs perfectly fine; however, when uploading the wheel and trying to use it in Databricks I get an error:

`ImportError: /python3.10/site-packages/parent/pyarmor_runtime_000000/pyarmor_runtime.so: undefined symbol: _PyCode_Validate`

The way I build the Wheel was:

- MANIFEST
```
recursive-include dist/parent/pyarmor_runtime_000000 *.so
```
- setup.cfg
```
[metadata]
    name = parent.child
    version = attr: parent.child.VERSION

[options]
    package_dir =
        =dist/

    packages =
        parent
        parent.child.innerchild
        parent.child.innerchild_2
        parent.child.innerchild_3
        parent.child.innerchild_4
        parent.child.innerchild_5
        parent.child.innerchild_6.innerchild
        parent.child.innerchild_7
        parent.pyarmor_runtime_000000

    include_package_data = True
```
- pyproject.toml
```
[build-system]
    requires = [
        "setuptools>=66.1.1",
        "wheel"
    ]
    build-backend = "setuptools.build_meta"
```

Obfuscation Command:
`pyarmor gen --recursive -i src/parent`

Build Command
`python -m build --skip-dependency-check --no-isolation`

Databricks Python is 3.10 so I'm building using a 3.10 Python venv.

Any clue on what might be happening?

**Comment by jondy** [comment]
First check this
https://pyarmor.readthedocs.io/en/latest/reference/solutions.html#id4

Generally, it's Python version issue
